include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -10422,6 +10877,13 @@
           "capabilities" : [ "PhysicalStorageBufferAddresses" ],
           "version" : "1.5"
         },
+        {
+          "enumerant" : "HitObjectAttributeNV",
+          "value" : 5385,
+          "extensions" : [ "SPV_NV_shader_invocation_reorder" ],
+          "capabilities" : [ "ShaderInvocationReorderNV" ],
```

Comment:
Think you should drop the extensions entry here too.

Reply:
I can, but seems we have extensions present for others (like ShaderRecordBufferKHR)
Are we expected to not have any extensions?

Reply:
It's the same issue David pointed out earlier in https://github.com/KhronosGroup/SPIRV-Headers/pull/300#discussion_r1014039717

If there is a capability, we don't also need the extension listed.